rotation.go

```diff
@@ -585,50 +572,3 @@ func (b *backend) popFromRotationQueueByKey(name string) (*queue.Item, error) {
 	}
 	return nil, queue.ErrEmpty
 }
-
-// getAllStaticRoles returns a slice of all static roles from storage
-func (b *backend) getAllStaticRoles(ctx context.Context, s logical.Storage) ([]string, error) {
```

Comment:
This was refactored to the generic `walkStoragePath` to be used for either roles or sets